CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Clean service and subservice domain fields when clean domain (#1201)
```

Comment:
```suggestion
- Fix: clean service and subservice domain fields when clean domain so logs now show correct serv= and subserv=  (#1201)
```

Reply:
I hope that!

Reply:
Fixed in [5e95c8d](https://github.com/telefonicaid/iotagent-node-lib/pull/1202/commits/5e95c8d5b6cb945f6e679f63c5993caa9af637a8)